docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -0,0 +1,169 @@
+---
+id: index
+title: "Zeebe Gateway"
+sidebar_label: "Overview"
+description: "The Zeebe Gateway is a component of the Zeebe cluster."
+---
+
+## What is the Zeebe Gateway?
+
+The Zeebe Gateway is a component of the Zeebe cluster; it can be considered the contact point for the Zeebe cluster which allows Zeebe clients to communicate with Zeebe brokers inside a Zeebe cluster. For more information about the Zeebe broker, visit our [additional documentation](../../components/zeebe/technical-concepts/architecture.md#brokers).
+
+To summarize, the Zeebe broker is the main part of the Zeebe cluster, which does all the heavy work like processing, replicating, exporting, and everything based on partitions. The Zeebe Gateway acts as a load balancer and router between Zeebe’s processing partitions.
+
+![Zeebe gateway overview](assets/zeebe-gateway-overview.png)
+
+To interact with the Zeebe cluster, the Zeebe client sends a command as a gRPC message to the Zeebe Gateway (to port `26500` by default). Given the gateway supports gRPC, the user can use several clients in different languages to interact with the Zeebe cluster. For more information, read our [overview](../../apis-clients/overview.md).
+
+When the Zeebe Gateway receives a valid gRPC message, it is translated to an internal binary format and forwarded to one of the partition leaders inside the Zeebe cluster. The command type and values can determine to which partition the command is forwarded.
```

Comment:
Hey @christinaausley, first thanks for the quick review! 

@berndruecker had one interesting comment in the gdoc regarding mentioning the partition leaders and going into RAFT details here, which might distract the normal reader. Do you have any thoughts on that?

Do you think it makes sense to add a note before that paragraph that says: :dragon: Be aware Zeebe brokers are using RAFT to replicate data and the data is shared into partition you can read more about that here https://docs.camunda.io/docs/components/zeebe/technical-concepts/clustering/#raft-consensus-and-replication-protocol

Reply:
Are you fine with merging this PR and then I will do the enhancements in a separate PR?

Reply:
Absolutely. Feel free to merge unless you need an additional review from @npepinpe.